apps/src/templates/sectionProgressV2/ItemType.jsx

```diff
@@ -3,7 +3,6 @@ import color from '@cdo/apps/util/color';
 
 export const ITEM_TYPE = Object.freeze({
   NOT_STARTED: 1,
-  VIEWED: 2,
```

Comment:
I am not sure if we should re-number `NEEDS_FEEDBACK` or `FEEDBACK_GIVEN` - I suppose we can... thoughts?

Also, I did a search for `ITEM_TYPE.VIEWED` to make sure we modified anywhere we had this still

Reply:
If we do plan to implement the "viewed" state in the relatively near future I'd probably lean toward leaving in the definition of the VIEWED state in this file, and the rendering logic in `ProgressIcon` and just take out the references to it in the icon key and popup window. Might be worth asking Bryan what the plan is here.

If we do want to delete `ITEM_TYPE.VIEWED` entirely, there's no harm in renumbering them, or in keeping 2 unused. Your call.

Reply:
@dju90 any thoughts on the future of the `VIEWED` state/icon?  In this PR I removed any reference to it, but if we think we might implement it in the future (as Eric pointed out), we should just not render it for now - which will make it easier to add it back in the future.  Do you have an idea of the future for this icon/state?

Reply:
We might implement it in the future, though it's unclear when. During prototype testing, users found it intuitive and valuable. However, I'm not sure yet which iteration will include this, as it does require backend changes to support it, so it's a relatively more costly investment.

Reply:
Thanks @dju90 and nice catch @etaderhold .  I made the changes to reflect this.